src/containers/Downloads/DownloadDatasetSummary.js

```diff
@@ -14,33 +50,8 @@ const DownloadDatasetSummary = () => {
             </tr>
           </thead>
           <tbody>
-            <tr className="downloads__table-row">
-              <td className="downloads__table-cell">Humans</td>
-              <td className="downloads__table-cell downloads__table-cell--value">
-                100
-              </td>
-              <td className="downloads__table-cell downloads__table-cell--value">
-                10
-              </td>
-            </tr>
-            <tr className="downloads__table-row">
-              <td className="downloads__table-cell">Mouse</td>
-              <td className="downloads__table-cell downloads__table-cell--value">
-                100
-              </td>
-              <td className="downloads__table-cell downloads__table-cell--value">
-                10
-              </td>
-            </tr>
-            <tr className="downloads__table-row">
-              <td className="downloads__table-cell">Total</td>
-              <td className="downloads__table-cell downloads__table-cell--value">
-                100
-              </td>
-              <td className="downloads__table-cell downloads__table-cell--value">
-                10
-              </td>
-            </tr>
+            {renderSpeciesRow()}
+            {renderTotalsRow()}
```

Comment:
I like this refactor! Maybe it would be good to extract these functions into separate components, or use the class notation for `DownloadDatasetSummary`. Otherwise `renderSpeciesRow` and `renderTotalsRow` will be redefined on every render.